### DISCUSSION TITLE
关于rrocr过验证码

### DISCUSSION THREAD

**Discussion by YourComputer011** [mikumifa/biliTickerBuy#117] [提问专区]
如果将多个（大约二十个）过验证码的请求，同时通过同一个api_key发送给云端，不知道能否同时处理？（能力有限，实测2个账号可同时处理）
关注这个问题的原因是：1、电脑上会有多个账号同时抢，可能会需要同时过验证码
                                        2、rrocr过验证码支持人工和机器一起过，但是人工完成后机器仍然会不停尝试该验证码，如果无法同时处理则会大幅延缓时间

以及rrocr网站的客服QQ号不通过好友55

**Comment by mikumifa** [comment]
你是黄牛吗，怎么能能做到要抢这么多的

**Reply by Coin-Lu** [reply]
艹，我也刚想问

**Reply by YourComputer011** [reply]
哈哈不是，六个人每个人手里都有三四个号，要是能少买一个服务就少买一个了

**Reply by mikumifa** [reply]
建议直接实践一下就好了，

**Reply by YourComputer011** [reply]
主要是其他人都在考试周没人理我ww，所以实践的话可能要等开票前一两天了w，我想的是前期能排查掉一些问题就先排查掉来着w

**Reply by mikumifa** [reply]
我也在复习考试，你们到时候抢票时候我还不在国内，到时候也不能帮你们补bug

**Comment by WittF** [comment]
你可以对着接口打点cc过去测压试试看过码还稳不稳定 (
